### ISSUE TITLE
LocationExtensions.ios.tvos.watchos.macos.cs file Contains wrong __iOS__ define

### ISSUE THREAD

**Issue by tpitman** [xamarin/Essentials#1856]
### Description
LocationExtensions.ios.tvos.watchos.macos.cs file Contains wrong __iOS__ define

### Steps to Reproduce

1. Check the default compiler settings on a Xamarin.iOS project when creating a cross platform Xamarin.Forms project
2. Notice the define in the compiler options is __IOS__ (capitol I)
3. Go to the LocationExtensions.ios.tvos.watchos.macos.cs file and notcie the __iOS__ define around the speed parameter has a lower case i instead of upper case.

### Expected Behavior
Include setting of the course and speed values of the Location object.

### Actual Behavior
Setting of the course and speed is NOT included.

### Basic Information

- Version with issue: 1.6.1
- Last known good version: ?
- IDE: VSMac 2019
- Platform Target Frameworks:
  - iOS:
- Nuget Packages:
- Affected Devices:
All

**Comment by jamesmontemagno** [comment]
Good catch, PR in :)

**Comment by jamesmontemagno** [comment]
I assume that this will fix #1855 ?

**Comment by tpitman** [comment]
It does not fix it, but does make it so I don't care. You see I only went down the road of trying to calculate my own GPS speed because this wasn't working. Now that this is working I can just use the GPS speed coming from the OS.

It might still be good to look at the other because something is not right somewhere. It could totally be the OS's fault because as far as I can tell the code just passes the CLLocation along.

The weird thing is I can't understand why it doesn't work. I have tried several different calculations and they all return similar results.

I wouldn't spend too much time on it though. It would just be interesting to have another set of eyes to at least verify there isn't something weird going on with the Xamarin code.
